applications/nrf_desktop/src/hw_interface/leds.c

```diff
@@ -193,8 +247,26 @@ static bool event_handler(const struct event_header *eh)
 
 		__ASSERT_NO_MSG(event->led_id < CONFIG_DESKTOP_LED_COUNT);
 
+		if (streaming && !event->stream) {
+			leds_state_bck[event->led_id] = event->led_effect;
+			return true;
```

Comment:
Maybe we should consume event always or never (no matter if it's stream event or not)?

Reply:
done